src/data/set/lattice.lean

```diff
@@ -701,12 +701,12 @@ theorem bUnion_union (s t : set α) (u : α → set β) :
   (⋃ x ∈ s ∪ t, u x) = (⋃ x ∈ s, u x) ∪ (⋃ x ∈ t, u x) :=
 supr_union
 
-@[simp] lemma Union_coe_set {α β : Type*} (s : set α) (f : α → set β) :
-  (⋃ (i : s), f i) = ⋃ (i ∈ s), f i :=
+@[simp] lemma Union_coe_set {α β : Type*} (s : set α) (f : s → set β) :
+  (⋃ i, f i) = ⋃ i ∈ s, f ⟨i, ‹i ∈ s›⟩ :=
```

Comment:
This change is a bit weird, but it matches `Union_subtype` so I'll believe you on that one.